src/components/pages/Profile/EditProfile.js

```diff
@@ -214,4 +215,11 @@ function EditProfile(props) {
     </>
   );
 }
-export default EditProfile;
+
+const mapStateToProps = state => {
+  return {
+    userInfo: state.user.userInfo,
+  };
+};
+
+export default connect(mapStateToProps)(EditProfile);
```

Comment:
I believe this is part of another PR you are working on. I'm not sure how to remove it from this PR.